### ISSUE TITLE
Contact Request: Rename "Discard" to "Not now"

### ISSUE THREAD

**Issue by foss-** [friendica/friendica#9714]
New contact request dialog has option to `Discard`. It is unclear to users what this button does and there is no way to learn about that via the UI. Proposing to changed that button to `Not now` as that is both less technical and more descriptive.

Not now: The request is rejected but the contact can send another request anytime.

While looking at this was wondering if `Approve` should be `Accept` to be less formal. 

![contact request](https://user-images.githubusercontent.com/5708172/103204408-99d61c00-48f7-11eb-98ac-f7577a428776.png)

**Comment by bkil** [comment]
Rewording `Ignore` to `Block` may also make it more in line with the competition.

**Comment by MrPetovan** [comment]
I agree, can you please [weigh in the issue #4960](https://github.com/friendica/friendica/issues/4960) on what `Ignore` should imply in terms of Friendica behavior ?
